AUTHORS.md

```diff
@@ -37,6 +37,7 @@ Programmers
     Britt Mathis (galdor557)
     Capostrophic
     cc9cii
+	Cédric Mocquillon
```

Comment:
Is this a tab? Usually we use 4 space characters instead.

Reply:
Yeah, convert to 4 spaces then I'll merge this. Thank you! :)

Reply:
Sorry for this, I just pushed the changes